### PR TITLE
Per project versioning

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

**Comment by LefterisJP** [comment]
won't work with split repositories

**Comment by arkpar** [comment]
https://github.com/ethereum/webthree-helpers/pull/18
https://github.com/ethereum/libweb3core/pull/6
https://github.com/ethereum/libethereum/pull/19
https://github.com/ethereum/mix/pull/75
https://github.com/ethereum/solidity/pull/79
https://github.com/ethereum/webthree/pull/41
https://github.com/ethereum/alethzero/pull/63
These PRs go together

File: libaleth/CMakeLists.txt [hunk 1]
```diff
@@ -15,8 +15,8 @@ set(EXECUTABLE aleth)
 add_library(${EXECUTABLE} STATIC ${SRC_LIST} ${JSRES} ${HEADERS} ${UI_RESOURCES})
 
 set_target_properties(${EXECUTABLE} PROPERTIES AUTOUIC ON AUTOMOC ON)
-eth_use(${EXECUTABLE} REQUIRED Qt::Core Qt::Widgets Qt::Network Web3::webthree Eth::devcore Eth::p2p Eth::devcrypto Eth::ethereum Web3::web3jsonrpc Eth::natspec)
+eth_use(${EXECUTABLE} REQUIRED Qt::Core Qt::Widgets Qt::Network Web3::webthree Eth::ethcore Eth::p2p Eth::devcrypto Eth::ethereum Web3::web3jsonrpc Eth::natspec)
 
-install( TARGETS ${EXECUTABLE} RUNTIME DESTINATION bin ARCHIVE DESTINATION lib LIBRARY DESTINATION lib )
-install( FILES ${HEADERS} DESTINATION include/${EXECUTABLE} )
+#install( TARGETS ${EXECUTABLE} RUNTIME DESTINATION bin ARCHIVE DESTINATION lib LIBRARY DESTINATION lib )
+#install( FILES ${HEADERS} DESTINATION include/${EXECUTABLE} )
```
Comment (debris) [inline libaleth/CMakeLists.txt]: Please remove these 2 lines

**Comment by debris** [comment]
looksgood :+1:

**Comment by arkpar** [comment]
tested to work with split repos, just need other PRs merged

**Pull request merged**